ui/src/testing/utils.js

```diff
@@ -39,3 +40,17 @@ export const reduceInitialState = (array, key, match, newValues) => {
     return acc;
   }, []);
 };
+
+/**
+ * Fixes the error...
+ * Warning: An update to Foo inside a test was not wrapped in act(...).\
+ * https://github.com/enzymejs/enzyme/issues/2073
+ * @param {ReactWrapper} wrapper The wrapper output from the enzyme `mount` command.
+ * @returns {Promise} completion of wrapper update.
+ */
+export const waitForComponentToPaint = async (wrapper) => {
+  await act(async () => {
+    await new Promise((resolve) => setTimeout(resolve));
```

Comment:
I'm not totally sure adding a timeout is better than wrapping things in `act`, but maybe this can be improved at some stage.